src/constants/chainInfo.ts

```diff
@@ -25,7 +25,7 @@ interface BaseChainInfo {
   readonly nativeCurrency: {
     name: string // e.g. 'Goerli ETH',
     symbol: string // e.g. 'gorETH',
-    decimals: number // e.g. 18,
+    decimals: 18 // e.g. 18,
```

Comment:
I don't think this is a safe change--what's your reasoning for making it?

Reply:
This is the definition of an ethereum-native currency according to web3-react: https://github.com/Uniswap/web3-react/blob/main/packages/types/src/index.ts#L62.
We need it to inter-operate. All of our defined native currencies use `18`, so it's internally safe - it's just a narrowing.

Reply:
Not sure if we can rely on this for all chains we might deploy to.